### DISCUSSION TITLE
Vulnerability search

### DISCUSSION THREAD

**Discussion by alexr3** [DefectDojo/django-DefectDojo#7435] [Q&A] [answered]
Hello everyone!

Added "branch_tag" to the configuration, but in the "View Active Findings" tab, the search for the tag is not performed (dojo does not find it).

**Comment by manuel-sommer** [comment, marked as answer]
Hi @alexr3,

Could you please add an issue with the exact steps that you did and with which version? In "Open findings" you have the possibility to choose one or multiple filters regarding tags.
- Tag name contains 
- Has tags
- Tags 
- Tags (Test)
- Tags (Engagement)
- Tags (Product)
- Not Tags
- Not Tags (Test)
- Not Tags (Engagement)
- Not Tags (Product)

Did you try them out? 

I guess this discussion can be closed / marked as answered @mtesauro.

Best,

Manuel
